src/components/output.jsx

```diff
@@ -151,11 +160,26 @@ export default class CellOutput extends React.Component {
     const value = this.props.valueToRender
 
     for (const handler of handlers) {
-      if (handler.shouldHandle(value)) {
-        const resultElem = handler.render(value)
-        if (resultElem !== undefined) {
+      let shouldHandle = false
+      try {
+        shouldHandle = handler.shouldHandle(value)
+      } catch (error) {
+        console.log('An output handler threw an exception in shouldHandle')
+      }
+      if (shouldHandle) {
+        var resultElem = handler.render(value)
```

Comment:
linter nit: `let` not `var` :-)

i don't know why, but this lint error does show up in my editor but doesn't seem to alert in mt build terminal (I'm guess you had the same experience). Maybe the linter is misconfigured for `npm run start`.

Reply:
Yeah, I see that, too.  Created #353 to track that.